lib/ansible/module_utils/network/eos/config/static_routes/static_routes.py

```diff
@@ -0,0 +1,375 @@
+
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        commands = []
+        haveconfigs = []
+        vrf = get_vrf(want)
+        for h in have:
+            return_command = add_commands(h)
+            for command in return_command:
+                if vrf == None:
+                    if "vrf" not in command:
+                        haveconfigs.append(command)
+                else:
+                    if vrf in command:
+                        haveconfigs.append(command)
+        wantconfigs = set_commands(want, have)
+
+        removeconfigs = list(set(haveconfigs) - set(wantconfigs))
+        for command in removeconfigs:
+            commands.append("no " + command)
+        for wantcmd in wantconfigs:
+            commands.append(wantcmd)
+        return commands
+
+    @staticmethod
+    def _state_overridden(want, have):
+        """ The command generator when state is overridden
+
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        commands = []
+        haveconfigs = []
+        for h in have:
+            return_command = add_commands(h)
+            for command in return_command:
+                haveconfigs.append(command)
+        wantconfigs = set_commands(want, have)
+        idempotentconfigs = list(set(wantconfigs) - set(haveconfigs))
+        if not idempotentconfigs:
+            return idempotentconfigs
+        removeconfigs = list(set(haveconfigs) - set(wantconfigs))
+        for command in removeconfigs:
+            commands.append("no " + command)
+        for wantcmd in wantconfigs:
+            commands.append(wantcmd)
+        return commands
+
+    @staticmethod
+    def _state_merged(want, have):
+        """ The command generator when state is merged
+
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
+        """
+        return set_commands(want, have)
+
+    @staticmethod
+    def _state_deleted(want, have):
+        """ The command generator when state is deleted
+
+        :rtype: A list
+        :returns: the commands necessary to remove the current configuration
+                  of the provided objects
+        """
+        commands = []
+        if not want:
+            for h in have:
+                return_command = add_commands(h)
+                for command in return_command:
+                    command = "no " + command
+                    commands.append(command)
+        else:
+            for w in want:
+                return_command = del_commands(w, have)
+                for command in return_command:
+                    commands.append(command)
+        return commands
+
+def set_commands(want, have):
```

Comment:
is there any reason on why these functions are outside of the class? If you don't want for them to be around there it might come handy to add some file such as utils.py for them

Reply:
I call these procs in the unit test